src/WebUI/Components/ServicesComponent.scss

```diff
@@ -10,16 +10,14 @@
 .sc-col-header {
     color: $secondary-text;
     font-size: $fontSizeM;
+    min-width: 140px;
 }
 
 .sc-col-content {
     display: inline-flex;
     align-items: center;
     font-size: 16px;
-}
-
-.sc-wide-col {
-    width: 140px;
+    min-width: 140px;
```

Comment:
i was talking about the port column width in the getcolumns function, not here. here it will make every column to be of the size 140